packages/build/package.json

```diff
@@ -95,11 +96,13 @@
     "log-process-errors": "^8.0.0",
     "map-obj": "^5.0.0",
     "memoize-one": "^6.0.0",
+    "node-fetch": "^3.3.2",
```

Comment:
Oh, nice! Is it full web-platform `fetch` compatible? We only need it because of that (see https://github.com/netlify/blobs?tab=readme-ov-file#custom-fetch)
From my understanding it'll need https://github.com/alexghr/got-fetch.

Would love to not add one more dependency though, but as mentioned in another piece we can probably remove this if we move build requirements to be Node 18+ (which we already do in CLI and it's the version we run in buildbot)

Reply:
Ignoring this for now as `got` didn't quite work and we need to add a dependency regardless. We should be cleaning this up somewhat soon and stop supporting Node <18 in here anyway.